src/app/shell/main/result/result.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import { Store } from '@ngxs/store';
+import { Observable } from 'rxjs';
+import { EditResultView } from '../../../shared/result.actions';
+
+@Component({
+  selector: 'app-result',
+  templateUrl: './result.component.html',
+  styleUrls: ['./result.component.scss']
+})
+export class ResultComponent implements OnInit {
+  data$: Observable<Object[]>;
+  public selectedVal: string;
+
+  constructor(private store: Store) {
+    this.data$ = this.store.select(state => state.ShowData);
```

Comment:
Not obvious what are you going to reach by selecting observable in one component and passing to another. Let's discuss tomorrow